### DISCUSSION TITLE
Add support for Expo Router and React Native Web

### DISCUSSION THREAD

**Discussion by ansh** [47ng/nuqs#784] [Ideas]
Currently, there is no adapter for Expo Router and/or React Native Web

It would be great to support this framework!

**Comment by franky47** [comment]
I agree! Would you like to open a PR?

**Reply by ansh** [reply]
I am happy to if I can get some guidance on overall steps required for the implementation.

**Reply by franky47** [reply]
The One.js community adapter docs might be a good place to start, they showcase the (yet unstable) API for custom adapters (IIRC One.js is also based on top of Expo Router):

https://github.com/47ng/nuqs/blob/next/packages/docs/content/docs/community-adapters/onejs.mdx
